ui/app/components/app/gas-customization/gas-price-button-group/tests/gas-price-button-group-component.test.js

```diff
@@ -107,6 +111,8 @@ describe('GasPriceButtonGroup Component', function () {
     let wrappedRenderButtonResult
 
     beforeEach(function () {
+      sinon.restore()
```

Comment:
Nit: Conventionally this is placed in the `afterEach` hook. This is what the `sinon` docs recommend, and I've been slowly adding it in various test modules as I update them.

Reply:
`sinon.restore()` is called in the `afterEach` of the root describe block (near the top of the fall). This was added because I thought I needed to stub the `renderButtonContent` method just for this `renderButton` describe block. The need for the stub came from the `this.context` property not being available when directly calling `GasPriceButtonGroup.prototype.renderButton`. However, by instead calling `wrapper.instance().renderButton`,  `this.context` is available and the stub and `sinon.restore()` are not needed.

This change was made in 383915c75

Reply:
Ok - it's good now, but I still don't follow the need for this `sinon.restore()`call here even in that circumstance.

Even if you want to delay this being stubbed, there doesn't seem to be any utility in globally restoring all stubs on the line before.